core/src/main/java/org/lflang/generator/c/CGenerator.java

```diff
@@ -928,6 +928,7 @@ protected void copyTargetFiles() throws IOException {
       FileUtil.copyDirectoryContents(coreLib, dest, true);
     } else {
       FileUtil.copyFromClassPath("/lib/c/reactor-c/core", dest, true, false);
+      FileUtil.copyFromClassPath("/lib/c/reactor-c/plugin-defaults", dest, true, false);
```

Comment:
Perhaps these should be called modules, not plugins. A module can be part of a core, but "plugin" almost always suggests some peripheral function.

Reply:
I am mostly agnostic to the naming choices that we use here. Note that in this case we actually do want to support the use of tracing as a peripheral function that users can swap out.

One consideration is that at least early on, we may want some fluidity between what is a plugin vs. what is an internal API, and many things may start out as internal APIs and turn into plugins. If it were up to me, I would wait for a compelling motivation based on a specific, justifiably problematic confusion before spending too much time introducing additional distinctions or discussing terminologies.

Reply:
I should also add (before I reference this thread in another discussion) that if it were up to me, pretty much all of our code would consist, at least aspirationally, of "plugins". For me whether something is a plugin is less about how important or "peripheral" its functionality is and more about whether third parties can implement it and "plug it in" to everything else. I see pretty much entity (research group, grad student, company, etc.) in our little LF world as being kind of like a third party in the sense of having their own independent goals, use cases, quality standards, stability requirements, etc.